lib/screens/picture_details_screen/picture_details_screen.dart

```diff
@@ -22,14 +27,19 @@ class _PictureDetailsScreenState extends State<PictureDetailsScreen> {
   String animalName = "";
   String kingdomName = "";
   String animalDetails = "";
+  double dangerous = randomNumber;
```

Comment:
Does this mean we're just setting the danger level to a random value if we don't know specifically that it's dangerous?

Reply:
yeah. I was going to set it to the same low value but I think it would be better if they differed. What are your thoughts? The random values only range from 0.0 to 0.4

Reply:
If it's a random low value then I'd say it's fine for now